src/components/ec-modal/ec-modal.vue

```diff
@@ -156,10 +156,20 @@ function hasPositiveButton() {
 function hasNegativeButton() {
   return !!slots.negative;
 }
+
+// zIndex
+const zIndexStyle = computed(() => (props.zIndex ? { zIndex: props.zIndex } : null));
+
+// focus trap
+const focusTrapTarget = ref(null);
+const focusTrapOptions = {
+  immediate: true,
+  escapeDeactivates: props.isClosable,
+  clickOutsideDeactivates: props.isClosable,
+};
 function getFocusTrapOptions() {
   const options = {
-    escapeDeactivates: props.isClosable,
-    clickOutsideDeactivates: props.isClosable,
+    ...focusTrapOptions,
```

Comment:
If `focusTrapOptions` is used only on this place, I'd keep the options inline in this function.